app/lib/format-helpers.js

```diff
@@ -1,19 +1,48 @@
-const formatAddress = (address) => {
+/**
+ *
+ * @param {{
+ *    addressLine1: string;
+ *    addressLine2: string;
+ *    town: string;
+ *    postcode: string;
+ *    country: string;
+ * }} address
+ * @param {boolean} hideCountry
+ * @returns {*[]|null}
+ */
+const formatAddress = (address, hideCountry = false) => {
   if (!address) {
     return null
   }
 
   const parts = []
 
   Object.keys(address).forEach(key => {
-    if (address[key]) {
+    const checkCountry = key !== 'country' || hideCountry === false
```

Comment:
I find this var name a bit confusing. I would reckon it should be 'includeField' as it actually relates to fields other than country.